### ISSUE TITLE
Triggered State?

### ISSUE THREAD

**Issue by adamomg** [Xorso/pyalarmdotcom#8]
Hi! First thanks for this, I use it all the time in Home Assistant.

Is there a way to get a "triggered" state to go along with the armed-home, armed-away, and disarmed?

**Comment by lorddaren** [comment]
Do you mean like if a sensor was triggered?

**Comment by adamomg** [comment]
Yeah, like if the alarm itself has been triggered. Like Home-Away is set and a window is opened - the alarm is triggered.  Also, if we could get the state of the sensors, that would also be super useful as well.

At this point, I would take the component working again. A lot of my automatons are based on its state. Thank you for looking into fixing it!

**Comment by adamomg** [comment]
Hi @Xorso !

Do you think it would be possible to track the state of sensors? or if the alarm has been triggered?
